qemu/tests/blockdev_mirror_hotunplug.py

```diff
@@ -24,7 +19,7 @@ def wait_till_frontend_devices_deleted(self):
         """Wait till devices removed from output of query-block"""
         def _is_device_deleted(device):
             for item in self.main_vm.monitor.query("block"):
-                if device in item["qdev"]:
+                if device in item.get("qdev"):
```

Comment:
item.get("qdev") can return None, then we may hit TypeError exception with this code change.

The simplest way is to change item.get("qdev") to item.get("qdev", device), but it's hard to understand, so we should make it clear with comments that qdev can be absent while the device hotunplug is still in progress, in this situation, we will wait till all block info is removed